src/components/ha-sidebar.ts

```diff
@@ -395,18 +425,52 @@ class HaSidebar extends LitElement {
             <mwc-icon-button class="show-panel">
               <ha-svg-icon .path=${mdiPlus}></ha-svg-icon>
             </mwc-icon-button>
-          </paper-icon-item>`;
+          </ha-clickable-list-item>`;
         })}
         ${this._renderSpacer()}`
       : ""}`;
   }
 
-  private _renderDivider() {
-    return html`<div class="divider"></div>`;
+  private _renderUtilityPanels() {
+    const [, afterSpacer] = computePanels(
+      this.hass.panels,
+      this.hass.defaultPanel,
+      this._panelOrder,
+      this._hiddenPanels
+    );
+
+    // prettier-ignore
+    return html`
+      <mwc-list
+        @focusin=${this._listboxFocusIn}
+        @focusout=${this._listboxFocusOut}
+        @scroll=${this._listboxScroll}
+        @keydown=${this._listboxKeydown}
+        class="utility-panels"
+      >
+        ${afterSpacer.map((panel) =>
+          this._renderPanel(
+            panel.url_path,
+            panel.url_path === this.hass.defaultPanel
```

Comment:
We should check for `component_name` === `lovelace` here, and not for `defaultPanel`

Reply:
Current master compares to `defaultPanel`. Is it wrong in master too then?